libnavigation-router/src/test/java/com/mapbox/navigation/route/internal/onboard/MapboxOnboardRouterTest.kt

```diff
@@ -80,7 +80,7 @@ class MapboxOnboardRouterTest {
         every { isValue } returns false
         every { isError } returns true
         every { value } returns null
-        every { error } returns RouterError(FAILURE_MESSAGE, FAILURE_CODE)
+        every { error } returns RouterError(FAILURE_MESSAGE, FAILURE_CODE, REQUEST_CODE)
```

Comment:
Do we also want to log the `requestId` in `MapboxOnboardRouter#generateErrorMessage` or better wait to refactor that when we migrate to the `HybridRouter`?

Reply:
```suggestion
        every { error } returns RouterError(FAILURE_MESSAGE, FAILURE_CODE, REQUEST_ID)
```

Reply:
> Do we also want to log the requestId in MapboxOnboardRouter#generateErrorMessage or better wait to refactor that when we migrate to the HybridRouter?

`getRoute` of `Router` is not returning requested, so it no sense to log it. Let's do it when migrating to `RouterInterface`